### PR TITLE
feature(readme) write proper readme

### PR DESCRIPTION
**Pull request by geromegrignon**
deletes Angular CLI default Readme template
adds course presentation with image and link to SFEIR website
adds script commands
adds course projects
adds stack
adds course program

updates #96

### CODE REVIEW

**Comment by noelmace** [comment]
Ouaw you really did a good work here, thx. I'll really read everything tomorrow though.
At first sight, I think we could maybe need to had some more informations about how to use the project itself (start, build, test, checkout, etc...).

**Comment by geromegrignon** [comment]
Few bugs i need to fix : 
- several broken links because of a missing # 
- put "Calendrier ..." under the course logo
- few typo corrections

I'll add a "Commandes utiles/ Useful Commands" section with script commands tomorrow.

File: README.md [hunk 1]
```diff
@@ -1,31 +1,156 @@
...
+## Déroulement de la formation
+Un TP :
+
+  * un projet par exercice :
+`npm run client step01`
+  * un projet par solution :
+`npm run client step01solution`
+
+ 
+## Planning de la formation
```
Comment (noelmace) [inline README.md]: Please remove this part, it would be to much to maintain for now.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,156 @@
...
+## Stack technique
+
+* Angular
+* NGXS
+* NGRX
+* Angular Material
+* NodeJS
+* ExpressJS
+
+## Commandes utiles
```
Comment (noelmace) [inline README.md]: this is the most important part of the readme. Please replace ng by `npm run ng --` and add a short mention that we can use ng if we install the cli globally. Thx

File: README.md [hunk 3]
```diff
@@ -1,31 +1,156 @@
...
+![logo](https://www.sfeir.com/img/school/formations/Angular%20200.png)
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
+[Calendrier des formations, liste des formateurs et programme de formation](https://www.sfeir.com/formation/school/angular-200/)
 
-## Running unit tests
+## Table des matières
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+  * [Support de formation](https://github.com/Sfeir/angular2-200/blob/step-01/support.pdf)
```
[the remark this answers is not in the document]
Comment (geromegrignon) [reply README.md]: Not in master branch but in step24 one (i took it as an example about final content required).
Deleting it.

File: README.md (hunk 1 above)
Comment (geromegrignon) [reply README.md]: As i'm seeing 2 sections, are you talking about the planning or the script commands for steps?
I deleted script commands meanwhile but will make changes depending on your answer.

Comment (noelmace) [reply README.md]: The planning part. Thx

Comment (geromegrignon) [reply README.md]: i removed it and added back the script commands section i deleted meanwhile.

File: README.md [hunk 4]
```diff
@@ -1,31 +1,85 @@
...
 
-## Further help
 
-To get more help on the `angular-cli` use `ng --help` or go check out the [Angular-CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+## Déroulement de la formation
+
+  Un TP :
+
+  * un projet par exercice :
+    `npm run client step01`
```
Comment (noelmace) [inline README.md]: you forgot --
npm run client -- hands-on
npm run client -- hands-on-solution

File: README.md [hunk 5]
```diff
@@ -1,31 +1,85 @@
...
+* Angular Material
+* NodeJS
+* ExpressJS
+
+
+## Commandes utiles
+
+Ces commandes doivent être toutes suffixées par le nom du projet : `step01`
+
+  * `npm run ng build` - Compiles an Angular app into an output directory(dist)
```
Comment (noelmace) [inline README.md]: you forgot --

File: README.md [hunk 6]
```diff
@@ -1,31 +1,85 @@
...
+  Un TP :
+
+  * un projet par exercice :
+    `npm run client step01`
+   * un projet par solution :
+    `npm run client step01solution`
+
+
+
+## Liste des projets
```
Comment (noelmace) [inline README.md]: remove this section

**Pull request merged**